session/session.php

```diff
@@ -246,18 +246,19 @@ function session_unset() {}
  * @param callable $update_timestamp [optional]
  * @return bool true on success or false on failure.
  */
-function session_set_save_handler(callable $open, callable $close, callable $read, callable $write, callable $destroy, callable $gc, $create_sid, $validate_sid, $update_timestamp): bool {}
+function session_set_save_handler(callable $open, callable $close, callable $read, callable $write, callable $destroy, callable $gc, callable $create_sid = ?, callable $validate_sid = ?, callable $update_timestamp = ?): bool {}
```

Comment:
` = ?` isn't a valid PHP syntax here. Per discussion in https://github.com/php/php-src/pull/5366#issuecomment-611996429 it seems to me that ` = ?` should be either replaced with `= null` or omitted at all.
As you can see [here](https://github.com/php/php-src/blob/master/ext/session/session.stub.php#L46), default values are unknown: specifying this in our stubs using PHP syntax isn't possible & hardly would be beneficial. So omitting them seems to be the best way.